Backoffice/GenerateForm/Strategies/ContactStrategy.php

```diff
@@ -36,6 +36,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'label' => 'open_orchestra_backoffice.block.contact.signature',
             'constraints' => new NotBlank(),
         ));
+        $builder->add('maxAge', 'hidden');
+    }
+
+    /**
+     * @return array
+     */
+    public function getDefaultConfiguration()
+    {
+        return array(
+            'maxAge' => 0
```

Comment:
@alavieille : I think it could be good to merge this default with the parent one, as the parent default values can evolve with new parameters in the future. Maybe this process of merging specific defaults with generic defaults could be coded to avoid explicit merge on each strategies.